packages/astro/test/cli.test.js

```diff
@@ -24,12 +24,6 @@ describe('astro cli', () => {
 		expect(proc.stdout).to.include(pkgVersion);
 	});
 
-	it('astro build', async () => {
```

Comment:
This test fails periodically, not always. We have so many build tests now (including the smoke tests) that I think this test has outlived its usefulness.